tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -136,14 +136,16 @@ Verify Oath-Proxy Image Is fetched From CPaaS
     Verify Container Image      redhat-ods-applications     ${pod}      oauth-proxy
     ...     "registry.redhat.io/openshift4/ose-oauth-proxy:v4.8"
 
-Verify Pytorch And Tensorflow Can Be Spawned
-    [Documentation]    Check Cuda builds are complete and  Verify Pytorch and Tensorflow can be spawned
-    [Tags]    Sanity
+Verify That CUDA Build Chain Succeeds
+    [Documentation]    Check Cuda builds are complete. Verify CUDA (minimal-gpu),
+    ...    Pytorch and Tensorflow can be spawned successfully
+    [Tags]    Smoke
     ...       Tier1
-    ...       ODS-480  ODS-481
+    ...       ODS-316
```

Comment:
why removed the tag ODS-480 ODS-481 ?

Reply:
I've explained in the description, those tests doesn't exist in Polarion (or I couldn't find them)

Reply:
Correcting myself, test _ODS-481 - Rhods-948: Verify cuda is successful in psi clusters_ does exist, but it looks unnecessary having _ODS-316 Verify That CUDA Build Chain Succeeds_

@pablofelix what would you prefer me to do, to add the tag ODS-481 to this test or to remove it from Polarion?

Reply:
I would prefer to add the tag

Reply:
Done